ibm/mas_devops/roles/ibm_catalogs/tasks/install/main.yml

```diff
@@ -21,34 +21,52 @@
 # 2.2 Offline catalog
 # If this fails then it means we can't perform an airgap install using this catalog
 - name: Load CASE bundle versions for the offline catalog
-  when: airgap_install
+  when:
+    - airgap_install
   include_vars:
     file: "{{ role_path }}/../../common_vars/casebundles/{{ mas_catalog_version }}.yml"
 
 # If we are updating the catalog in an airgap environment, we first have to
 # update the image digest configmaps, otherwise when the operators update
 # they will not be able to use digests for the new images.
 - name: "Update digest maps ahead of catalog update"
-  when: airgap_install
+  when:
+    - airgap_install
+    - not dev_mode_airgap
```

Comment:
We shouldn't need this ... `update-digest-maps.yml` is already version specific:  For example MAS core .. when we call this it will do nothing when `version == 8.10.x`, or `8.11.x-dev` ... so we can always call the task, and rely on the existing logic in the task to do nothing in the pre-release airgap testing scenario (because we know that MAS Core 8.10+ has native digest support built in).

```
- name: "Update ibm-mas Image Digest Map (8.8.x)"
  when: "'8.8.x' is in mas_core_version"
  include_role:
    name: ibm.mas_devops.suite_install_digest_cm
  with_items: "{{ suites.resources }}"
  vars:
    digest_image_map_namespace: "{{ item.metadata.namespace }}"
    case_name: ibm-mas
    case_version: "{{ mas_core_version['8.8.x'] }}"

- name: "Update ibm-mas Image Digest Map (8.9.x)"
  when: "'8.9.x' is in mas_core_version"
  include_role:
    name: ibm.mas_devops.suite_install_digest_cm
  with_items: "{{ suites.resources }}"
  vars:
    digest_image_map_namespace: "{{ item.metadata.namespace }}"
    case_name: ibm-mas
    case_version: "{{ mas_core_version['8.9.x'] }}"
```